src/test/scala/chisel3/tests/AsyncResetNativeTest.scala

```diff
@@ -0,0 +1,171 @@
+// See LICENSE for license details.
+
+package chisel3.tests
+
+import chisel3._
+import chisel3.experimental.MultiIOModule
+import chisel3.tester._
+import chisel3.tester.experimental.TestOptionBuilder._
+import chisel3.tester.internal.VerilatorBackendAnnotation
+import firrtl.AnnotationSeq
+import org.scalatest._
+
+/**
+  * circuit that illustrates usage of async register
+  * @param resetValue value on reset
+  */
+class NativeAsyncResetRegModule(resetValue: Int) extends MultiIOModule {
```

Comment:
I don't think it makes sense to mark this as Native, that should be implicit and default. If we have another AsyncReset test that eg uses blackboxes, that should be marked as special.

Reply:
done